### ISSUE TITLE
[BUG] TF2BD Crashes when you click "Launch"

### ISSUE THREAD

**Issue by Phencyclidine08** [PazerOP/tf2_bot_detector#342]
When I updated to this new version of TF2BD it crashes upon clicking launch. I reversed the launch option changes needed for the temp fix and everything, it still crashes, even on a fresh install with the default cfg files.

**To Reproduce**
Install latest version of TF2BD
Run TF2BD
Click "Launch"
Crash

I'm using Windows 10
TF2 Bot Detector V1.2.1.934

I will provide a mdmp crash dump file if that helps.
[https://puu.sh/HSmoK/e7417dd925.mdmp](url)

**Comment by PazerOP** [comment]
Try nightly build 1.2.1.939 when you have a chance and see if it fixes the crashes. The tool might be failing to determine your command line arguments for some reason, so you'll probably see some error messages in the bottom right pane once tf2 opens.

**Comment by Phencyclidine08** [comment]
> Try nightly build 1.2.1.939 when you have a chance and see if it fixes the crashes. The tool might be failing to determine your command line arguments for some reason, so you'll probably see some error messages in the bottom right pane once tf2 opens.

Could I get a link to that? I'm not seeing it in the asset list for 1.2.1.939

**Comment by ClusterConsultant** [comment]
https://github.com/PazerOP/tf2_bot_detector/suites/3156248596/artifacts/72585621

**Comment by ClusterConsultant** [comment]
@Phencyclidine08 is this still an issue?

**Comment by Phencyclidine08** [comment]
> @Phencyclidine08 is this still an issue?

Yes. This is still occurring on the latest version. That nightly build links to a 404 error. I'd still like some help with this.

**Comment by Phencyclidine08** [comment]
@ClusterConsultant ^^^ Forgot to ping you in my reply.
